src/components/TextPicker/TextSelectorModal.tsx

```diff
@@ -59,12 +69,13 @@ function TextSelectorModal({value, description = '', onValueSelected, isVisible,
             >
                 <HeaderWithBackButton
                     title={description}
-                    onBackButtonPress={onClose}
+                    onBackButtonPress={hide}
                 />
                 <ScrollView
                     contentContainerStyle={[styles.flex1, styles.mh5, styles.mb5]}
                     keyboardShouldPersistTaps="handled"
                 >
+                    <View style={styles.pb4}>{!!subtitle && <Text style={StyleUtils.combineStyles([styles.sidebarLinkText, styles.optionAlternateText, styles.pre])}>{subtitle}</Text>}</View>
```

Comment:
Do we need really to use `StyleUtils.combineStyles()` here? Have you tested without it?